troubleshooting/known_issues/ims_soft_deleted_recipe_always_has_require_dkms_true.md

```diff
@@ -0,0 +1,38 @@
+# Soft Deleted IMS Recipe Always Has `require_dkms=true`
+
+## Issue description
+
+When an IMS recipe is deleted and becomes a "deleted recipe", its `require_dkms` attribute is set to `true`,
+regardless of its original value. This new value persists even if the recipe is later restored using an `undelete`
+operation.
+
+## Resolution
+
+(`ncn-mw#`) The problem can be worked around by manually updating the `require_dkms` value after the deleted
+has been restored.
+
+> In the following command, replace `<RECIPE_ID>` with actual IMS recipe ID.
+
+```bash
+cray ims recipes update --require-dkms false <RECIPE_ID> --format json
+```
+
+Example output:
+
+```json
+{
+  "arch": "aarch64",
```

Comment:
This output does not look like it was from CSM 1.4. I only mention it because we should verify that `arch` was a possible field in IMS recipes in CSM 1.4. I don't remember when it was added.